<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>readlog</class>
 <widget class="QWidget" name="readlog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_set">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>üëáËØ¶ÁªÜËÆæÁΩÆ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reload">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>40</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ÈáçÊñ∞Âä†ËΩΩ</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_set">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>170</y>
     <width>411</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>ËÆæÁΩÆ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>80</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ÂÖ≥Èó≠</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>LogÁõÆÂΩï</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_folder">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
